src/community/security/keycloak/pom.xml

```diff
@@ -48,7 +48,6 @@
 		<dependency>
 			<groupId>net.sf.json-lib</groupId>
 			<artifactId>json-lib</artifactId>
-			<classifier>jdk15</classifier>
```

Comment:
Is this related with this fix?

Reply:
Indeed, it's related to json-lib fork new release version.